streaq/types.py

```diff
@@ -38,3 +55,58 @@ class WrappedContext(Generic[WD]):
     tries: int
     ttl: timedelta | int | None
     worker_id: str
+
+
+Middleware: TypeAlias = Callable[
+    [WrappedContext[WD], Callable[..., Coroutine]], Callable[..., Coroutine]
+]
+
+
+CronTaskFn: TypeAlias = Callable[[WrappedContext[WD]], Coroutine[Any, Any, R] | R]
```

Comment:
Let's group all the TypeAliases together, without extra line spacing.

Reply:
Done